### DISCUSSION TITLE
ToDoListのスワイプで削除機能を作りたいです。

### DISCUSSION THREAD

**Discussion by KOTARO1209** [lit-iphone-course/lit-iphone-course#142] [Question-original]
## (1) 実現したいこと
スワイプで削除する機能を実装したいです。
## (2) 実現するために自分が試した内容とその結果
①スワイプすると削除アイコンが出る、②そのアイコンを押すとsaveData、titles、contentsの要素を削除する、の2つに分けて実装しようと考えました。
検索してまずは「スワイプすると削除アイコンが出る」まで実装しようとしましたが、スワイプしてもそもそも表示されないです。
## (3) 調べた内容や資料
https://ryolog.org/uitableview-delete/
## (4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
![スクリーンショット 2023-02-21 0 12 27](https://user-images.githubusercontent.com/75009903/220143388-18b1e664-9c9a-4a68-833b-9d9b4ee14b69.png)
## (5) GitHubのリンク
https://github.com/KOTARO1209/ToDo

**Comment by appleEmily** [comment]
今、CollectionViewを使ってやってて、でもスワイプして削除はtableViewだから、そのせいでコードを書いてももちろん反映されないっていうおかしいことになってる〜
スワイプして削除の実装するならtableViewを使おう！

**Reply by KOTARO1209** [reply]
ありがとうございます🙇‍♂️
tableViewで再度やってみます！
